Update avatar-standards.md (#171)
vircadia/vircadia-docs-sphinx: docs/source/create/avatars/avatar-standards.md

    bs = LipsUpperClose = head_BS_mouth_down = 0.1  <br />
</div>

**ReadyPlayerMe Blendshapes**

> ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically using a default mapping.

Reviewer: ```suggestion
ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically.
```
The average user won't care how this is done, and the additional info doesn't help technical users.

**Unsupported Blendshapes**

> Utilizing Unsupported blendshapes present on models or avatars must match one of the blendshapes below. Alternatively, the utilization of a .fst file can be used in order to remap the blendshapes into any of the ones below.

Reviewer: ```suggestion
To utilize blendshapes present on avatars, they must match the blendshapes below. Alternatively, an .fst file can be used to remap the blendshapes to any of the ones below.
```

> **ReadyPlayerMe Blendshape**

Reviewer 2: change to **ReadyPlayerMe Blendshapes**

> ReadyPlayerMe blendshapes are converted into Vircadia blendshapes automatically using a default mapping

Reviewer 2: It may be unclear if you don't know about RPM avatars, perhaps explain a bit more?

The default blendshapes used in [ReadyPlayerMe](https://readyplayer.me/) avatars are converted into Vircadia blendshapes automatically using a default mapping

> **Eye Blendshapes**

Reviewer 2: I would suggest a starting paragraph explaining, something along the lines:
### Supported Blendshape names
I order to drive a Blendshape (on avatars or models) it has to either match one of the names in the list below, or use an *.fst file to remap it to one of the names in the list as described above.

Author: I've included an additional section titled Unsupported Blendshapes containing the information you suggested to include in the documentation.
